estate/models/estate_property.py

```diff
@@ -0,0 +1,153 @@
+from odoo import api, fields, models, exceptions, tools
+from dateutil.relativedelta import relativedelta
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char(
+        "Title", required=True,
+        help="This field specifies the estate property name.",
+    )
+    description = fields.Text(
+        "Description",
+        help="This field specifies the description of property in brief to provide insights on the property.",
+    )
+    postcode = fields.Char(
+        "Postcode", help="This field specifies the postcode of the property address."
+    )
+    date_availability = fields.Date(
+        "Available From", readonly=False, default=fields.Date.today() + relativedelta(months=3),
+        help="This field specifies the date when the property will be available.",
+    )
+    expected_price = fields.Float(
+        "Expected Price", required=True,
+        help="This field specifies the price expected for the property.",
+    )
+    selling_price = fields.Float(
+        "Selling Price", readonly=True, copy=False,
+        help="This field specifies the actual selling price of the property.",
+    )
+    bedrooms = fields.Integer(
+        "Bedrooms", default=2,
+        help="This field specifies the number of bedroom that this property consists of.",
+    )
+    living_area = fields.Integer(
+        "Living Area (sqm)",
+        help="This field specifies the size of living area (in sqm.) of the property.",
+    )
+    facades = fields.Integer(
+        "Facades",
+        help="This field specifies the facade i.e. the exterior or the front of a property.",
+    )
+    garage = fields.Boolean(
+        "Garage",
+        help="This field specifies if the property has a garage",
+    )
+    garden = fields.Boolean(
+        "Garden",
+        help="This field specifies if the property has a garden",
+    )
+    garden_area = fields.Integer(
+        "Garden Area (sqm)",
+        help="This field specifies the size of garden area (in sqm.) of the property.",
+    )
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+        help="This selection fields specifies the facing direction of garden (North, South, East, or West).",
+    )
+    state = fields.Selection(
+        string="Property Status", copy=False, required=True, default="new",
+        selection=[
+            ("new", "New"),
+            ("offer received", "Offer Received"),
+            ("offer accepted", "Offer Accepted"),
```

Comment:
Always use ``_`` , it should not contain spaces.
```suggestion
            ("offer_received", "Offer Received"),
            ("offer_accepted", "Offer Accepted"),
```